src/modules/85-cv/components/Notifications/NotificationsContainer.utils.ts

```diff
@@ -147,6 +145,21 @@ export function validateNotificationConditions(
     if (!condition?.condition && !values?.conditions?.[index]?.condition) {
       set(errors, `conditions.${index}.condition`, `${getString('cv.notifications.validations.conditionIsRequired')}`)
     }
+    if (
+      condition?.condition?.value == Condition.CODE_ERRORS ||
+      values?.conditions?.[index]?.condition == Condition.CODE_ERRORS
+    ) {
+      if (!condition?.eventStatus?.length && !values?.conditions?.[index]?.eventStatus?.length) {
+        set(
+          errors,
+          `conditions.${index}.eventStatus`,
+          `${getString('cv.notifications.validations.eventStatusIsRequired')}`
```

Comment:
No need for the template string here
```suggestion
          getString('cv.notifications.validations.eventStatusIsRequired')
```